components/lib/password/PasswordBase.js

```diff
@@ -85,6 +85,7 @@ export const PasswordBase = ComponentBase.extend({
         header: null,
         content: null,
         footer: null,
+        iconField: null,
```

Comment:
I don't think you need this we aren't adding a new property just a new passthrough.

Reply:
Without this it wont work 'React does not recognize the `iconField` prop on a DOM'

Reply:
I just updated your PR can you do a git pull and test again with my changes?

Reply:
Okey